skyportal/plot.py

```diff
@@ -2329,12 +2329,36 @@ def make_spectrum_layout(obj, spectra, user, device, width, smoothing, smooth_nu
         width=slider_width,
         margin=(0, 10, 0, 10),
     )
+    w_title = Div(text="Custom wavelength")
+    w_input = NumericInput(
+        value=0.0,
+        mode='float',
+    )
 
     # Track elements that need to be shifted with change in z / v
     shifting_elements = []
     renderers = []
     obj_redshift = 0 if obj.redshift is None else obj.redshift
 
+    flux_values = list(np.linspace(ymin, ymax, 100))
+    flux_values[-1] = np.nan
```

Comment:
Putting NaN in the last value makes the whole line disappear?

Reply:
I'm not sure where `flux_values` is used?

Reply:
I copied and pasted what was below to make it work but I ended up removing flux_values from el_data, so I should move it back where it was, above the for loop line 2362.

Reply:
OK I guess I just had to expand the view a little more to see it. 
Also I'm not clear on what a NaN on the last value does.